cmd/agent/dist/conf.d/processes.d/conf.yaml.default

```diff
@@ -0,0 +1,2 @@
+instances:
```

Comment:
we already have a `process` check, can we call this `winproc` or something like this?